### PR TITLE
cassandra: Added amazing cassandra configuration

### PR DESCRIPTION
**Pull request by hannesrabo**

### CODE REVIEW

File: cassandra_slo/mininet/topology_slo.py [hunk 1]
```diff
@@ -50,9 +56,23 @@ def __init__(self):
         self.startCassandra('h1')
         self.startCassandra('h2')
 
+    # TESTING CORE PINNING (returns number of cores)
+	numOfCores = numCores()
+	print 'The number of cores is: ', numOfCores
+
+    # Assign cores to the hosts
+	self.getHost('h1').config(cores=1)
```
Comment (juliuscc) [inline cassandra_slo/mininet/topology_slo.py]: @ShubhankerS is anything missing from here? Hannes says that he added the things from the server. Please make sure that nothing is missing.

Ping @PethrusG also.

File: cassandra_slo/mininet/topology_slo.py [hunk 2]
```diff
@@ -73,7 +93,7 @@ def createTopology(self):
                              privateDirs=cassandra_private_dirs)
         host6 = self.addHost('h6', ip="100.0.0.16/24",
                              privateDirs=cassandra_private_dirs)
-
+
```
Comment (juliuscc) [inline cassandra_slo/mininet/topology_slo.py]: Is there a good reason for this?

File: cassandra_slo/mininet/topology_slo.py (hunk 1 above)
Comment (hannesrabo) [reply cassandra_slo/mininet/topology_slo.py]: This has been removed.

File: cassandra_slo/mininet/topology_slo.py (hunk 2 above)
Comment (hannesrabo) [reply cassandra_slo/mininet/topology_slo.py]: This has been resolved.

**Pull request merged**